okio/src/main/java/okio/AsyncTimeout.java

```diff
@@ -49,15 +52,14 @@
   private static final int TIMEOUT_WRITE_SIZE = 64 * 1024;
 
   /**
-   * The watchdog thread processes a linked list of pending timeouts, sorted in
-   * the order to be triggered. This class synchronizes on AsyncTimeout.class.
-   * This lock guards the queue.
-   *
-   * <p>Head's 'next' points to the first element of the linked list. The first
-   * element is the next node to time out, or null if the queue is empty. The
-   * head is null until the watchdog thread is started.
+   * The watchdog thread processes a queue of pending timeouts, sorted in the order to be triggered.
    */
-  private static AsyncTimeout head;
+  private static class QueueHolder {
```

Comment:
good point. I didn't want to initialize it if the watchdog was never used. I added it to the watchdog initialization.